week_4/quiz-trivia-app/src/app/data/quiz.service.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable } from '@angular/core';
+import data from './data.json';
+import { Quiz } from './quiz.model';
+import { BehaviorSubject, Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class QuizService {
+  private selectedSubject = new BehaviorSubject<Quiz | undefined>(undefined);
```

Comment:
pass null instead of undefined and use only Quiz as type

Reply:
done